Server/src/BaseQuasarServer.cpp

```diff
@@ -305,11 +308,12 @@ int BaseQuasarServer::parseCommandLine(
     desc.add_options()
             ("config_file", value<string>(), "A path to the config file")
             ("opcua_backend_config", value<string>(&opcUaBackendConfigurationFile)
-	         ->default_value(defaultOpcUaBackendConfigurationFile),
+             ->default_value(defaultOpcUaBackendConfigurationFile),
                  "(Optional) path to the OPC-UA settings file")
             ("create_certificate", bool_switch(&createCertificateOnly), "Create new certificate and exit")
             ("help,h", "Print help")
-            ("version,v", bool_switch(&printVersion), "Print version and exit");
+            ("version,v", bool_switch(&printVersion), "Print version and exit")
+            ("extra_info,x", bool_switch(&printExtraInfo), "Print extra info and exit");
```

Comment:
My suggestion is not to steal "x" from all quasar servers. What about "-vx" (verbose extended) or "-vv" (more verbose?) ?

Reply:
It sounds good to use "vx", I will do that. 
On the other hand, I wouldn't go for "vv" since it has slightly different semantics in most linux tools.

Reply:
Hello @pnikiel, there is one issue to overcome with this suggestion. That is that boost program options allow for only 1 letter short-option.
One for example cannot write sth like
`("version_extra,vx", bool_switch(&printVersionExtraInfo), "Print version extra info and exit");`

We can of course avoid the short option as a whole, but I think it is convenient. For that, IMO "x" can be ok if there is no known server that uses it. What is your opinion?

Reply:
What if one considers "vx" to be long letter, i.e. "--vx" ? (**double dash**)

About "no known server that uses it" : I believe you can't be sure.

My opinion is not to do "-x" however I'd have strong preference for your strong involvement in decision making (for known reasons) so I can leave the final word to you.